spec/rspec/core/formatters/html_formatted.html

```diff
@@ -330,32 +331,61 @@
     <script type="text/javascript">makeRed('example_group_6');</script>
-    <script type="text/javascript">moveProgressBar('62.5');</script>
+    <script type="text/javascript">moveProgressBar('60.0');</script>
     <dd class="example failed">
       <span class="failed_spec_name">fails</span>
       <span class="duration">n.nnnns</span>
       <div class="failure" id="failure_2">
-        <div class="message"><pre>Failure/Error: expect(1).to eq(2)
-
-  expected: 2
-       got: 1
-
-  (compared using ==)</pre></div>
-        <div class="backtrace"><pre>./spec/rspec/core/resources/formatter_specs.rb:33:in `block (2 levels) in &lt;top (required)&gt;&#39;</pre></div>
-    <pre class="ruby"><code><span class="linenum">31</span><span class="constant">RSpec</span>.describe <span class="string"><span class="delimiter">&quot;</span><span class="content">failing spec</span><span class="delimiter">&quot;</span></span> <span class="keyword">do</span>
-<span class="linenum">32</span>  it <span class="string"><span class="delimiter">&quot;</span><span class="content">fails</span><span class="delimiter">&quot;</span></span> <span class="keyword">do</span>
-<span class="offending"><span class="linenum">33</span>    expect(<span class="integer">1</span>).to eq(<span class="integer">2</span>)</span>
-<span class="linenum">34</span>  <span class="keyword">end</span>
-<span class="linenum">35</span><span class="keyword">end</span></code></pre>
+        <div class="message"><pre>  Failure/Error: expect(1).to eq(2)
+
+    expected: 2
+         got: 1
+
+    (compared using ==)
+  # ./spec/rspec/core/resources/formatter_specs.rb:37:in `block (2 levels) in &lt;top (required)&gt;&#39;</pre></div>
+        <div class="backtrace"><pre>./spec/rspec/core/resources/formatter_specs.rb:37:in `block (2 levels) in &lt;top (required)&gt;&#39;</pre></div>
+    <pre class="ruby"><code><span class="linenum">35</span><span class="constant">RSpec</span>.describe <span class="string"><span class="delimiter">&quot;</span><span class="content">failing spec</span><span class="delimiter">&quot;</span></span> <span class="keyword">do</span>
+<span class="linenum">36</span>  it <span class="string"><span class="delimiter">&quot;</span><span class="content">fails</span><span class="delimiter">&quot;</span></span> <span class="keyword">do</span>
+<span class="offending"><span class="linenum">37</span>    expect(<span class="integer">1</span>).to eq(<span class="integer">2</span>)</span>
+<span class="linenum">38</span>  <span class="keyword">end</span></code></pre>
+      </div>
+    </dd>
+    <script type="text/javascript">moveProgressBar('70.0');</script>
+    <dd class="example failed">
+      <span class="failed_spec_name">fails twice</span>
+      <span class="duration">n.nnnns</span>
+      <div class="failure" id="failure_3">
+        <div class="message"><pre>  Got 2 failures:
+
+  .1) Failure/Error: expect(1).to eq(2)
```

Comment:
Oh, I see why: it's because the exception presenter expects a failure number (e.g. `2`), so that it would print `2.1` and `2.2`. 

I think we should fix it so that if the failure index is `nil` it prints `1)` and `2)` instead of `.1)` and `.2)`.